docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,628 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in Aggregate functions defined in the dataset API and SQL interface. Aggregate functions
+operate on a group of rows and return a single value.
+
+Spark SQL Aggregate functions are grouped as <code>agg_funcs</code> in spark SQL. Below is the list of functions.
+
+**Note:** Every below function has another signature which takes String as a column name instead of Column.
```

Comment:
`All functions below have another signature...`?

Reply:
@maropu Thanks for pointing this out, I will change All to Many. Notice that some of the new  aggregate functions don't have dataset API. Like `any/some/bool_or`

Reply:
Thanks, will make changes.